modules/Bio/EnsEMBL/Variation/DBSQL/PhenotypeFeatureAdaptor.pm

```diff
@@ -1627,4 +1708,59 @@ sub _get_submitter_name{
 
 }
 
+=head2 use_phenotype_classes
+
+  Arg [1]    : string $newval (optional)
+  Example    :
+    # Configure the adaptor to return all phenotype classes (including non specified)
+    # when using fetch methods in the various object adaptors
+    $pfa->use_phenotype_classes('trait,tumour,non_specified');
```

Comment:
Suggest a sub that returns what phenotype_classes can be use in list

Reply:
Thanks for the suggestion will add this in a next commit.

Reply:
Currently by default all are returned. If this will be prove popular then I'll add it.